bark/models/behavior/idm/idm_lane_tracking.hpp

```diff
@@ -32,6 +32,9 @@ class BehaviorIDMLaneTracking : public BaseIDM {
     crosstrack_error_gain_ =
         params->GetReal("BehaviorIDMLaneTracking::CrosstrackErrorGain",
                         "Tuning factor of stanley controller", 1.0);
+    dynamic::Input input(2);
+    input << 0.0, 0.0;
+    SetLastAction(input);
```

Comment:
why is this required in the constructor?

Reply:
Cause this https://github.com/bark-simulator/bark/blob/master/bark/models/behavior/idm/base_idm.cpp#L78 needs to be overwritten after the  BaseIDM constructor is called in the BehaviorIDMLaneTracking constructor